package.json

```diff
@@ -56,6 +56,7 @@
     "localforage": "^1.10.0",
     "mathjs": "^13.0.3",
     "monaco-editor": "^0.52.0",
+    "piexifjs": "^1.0.6",
```

Comment:
Forgot to mention, but this package addition should be in it's own commit as well, together with the `bun.lockb` update.